### DISCUSSION TITLE
Request Venus OS

### DISCUSSION THREAD

**Discussion by aukeboskma** [tteck/Proxmox#2258] [Ideas]
I'm looking for a script for Venus OS to run this for my battery managing systeem

https://github.com/victronenergy/venus

**Comment by SergiCEES** [comment]
If you get how to do it, please tag me here! ;) cheers

**Comment by brettwellmanmbo** [comment]
Also interested in this.  Two routes to go about this it seems:

1) VenusOS packages on top of a Debian VM. 
https://github.com/victronenergy/venus/wiki/install-venus-packages-on-Debian

2) Running an ARM64 VM and then getting the VenusOS raspberrypi distro installed on it.
https://dev.to/duelnm/setting-up-arm-vm-on-proxmox-ve-47a0
